src/main/java/com/novi/springboottechiteasy/controllers/CiModuleController.java

```diff
@@ -0,0 +1,71 @@
+package com.novi.springboottechiteasy.controllers;
+
+import com.novi.springboottechiteasy.dtos.cimoduledtos.CiModuleDto;
+import com.novi.springboottechiteasy.dtos.cimoduledtos.CiModuleInputDto;
+import com.novi.springboottechiteasy.models.CiModule;
+import com.novi.springboottechiteasy.services.CiModuleService;
+import jakarta.validation.Valid;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+import java.net.URI;
+import java.util.List;
+
+@RestController
+public class CiModuleController {
```

Comment:
Je kunt bovenaan je controller klasse de @RequestMapping annotatie zetten met als parameter "/ci-modules", dan hoef je dat niet meer bij elke afzonderlijke mapping te zetten en kunnen de paden van je mapping leeg blijven, of een enkele "/{id}" bevatten.